.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "content/acikkaynak"]
+	path = content/acikkaynak
+	url = https://github.com/acikkaynak/acikkaynak
```

Comment:
`git submodule` ile ana repodaki içeriklere `content/acikkaynak` üzerinden erişebiliyoruz. Fakat senkronizasyon için CI/CD'de bir düzenleme gerekebilir.